### PR TITLE
[docs] How to test in GitHub actions

### PR DESCRIPTION
**Pull request by esteban-url**
This PR creates a how to section on How to have Github actions test your redwood app

### CODE REVIEW

**Comment by keithtelliott** [comment]
@esteban-url please see the PR I submitted against your fork.  I performed basic edits and left a couple PR comments.  

General question:  What is the recommended way to contribute/edit files associated with somebody else's PR?  

Here's how I did it (as a noob that is figuring it out)...  Created my own fork of redwoodjs/redwood, checked out the test-in-github-actions branch, renamed it, edited the article, pushed it to my forked repo, then submit a PR to esteban-url.  

Seems a bit round-about, but I think it works.  I also tried to push commits directly (instead of creating a PR, but I got 403 errors).  

Thanks for helping me work through the GitHub flow!

**Comment by esteban-url** [comment]
@keithtelliott thanks a bunch! I check out your PR, and look into granting you the necessary access to my repo.

File: docs/docs/how-to/test-in-github-actions.md [hunk 1]
```diff
@@ -0,0 +1,394 @@
...
+
+<img width="1460" alt="ci-pr-1" src="https://user-images.githubusercontent.com/14810250/202825767-c7f23b24-e311-4a70-bf50-fbad40a6abee.png">
+
+Once the Github Action is done, you can see the results in the "Conversation" tab:
+
+<img width="1385" alt="ci-pr-2" src="https://user-images.githubusercontent.com/14810250/202825772-93c8fe50-6b91-4048-882b-21497d47e211.png">
+
+### 6. Deploy the database changes to an actual database
+
+Now we want to use another action to deploy the database changes to an actual database, so we can test the app in a real environment. In this action we will run the tests one more time against the local database, and then deploy the database migrations to the external database.
```
[the remark this answers is not in the document]
Comment (esteban-url) [reply docs/docs/how-to/test-in-github-actions.md]: Yes, I agree, the better word for it is run. thanks @virtuoushub

**Comment by keithtelliott** [comment]
@esteban-url I worked through the How To and ran my unit tests within GitHub Actions.  It just worked!  Your technical description successfully guided me through the required steps.  So, I did not add/adjust the technical details.  

I did, however, add a callout note mid-way through, and a summary conclusion at the end.  My goal:  encourage the reader.

**Comment by jtoar** [comment]
Looks like we're getting a build error; trying to find out what's causing it

image

File: docs/docs/how-to/test-in-github-actions.md [hunk 2]
```diff
@@ -0,0 +1,406 @@
...
+DATABASE_URL=postgres://postgres:postgres@localhost:54322/postgres
+TEST_DATABASE_URL=postgres://postgres:postgres@localhost:54322/postgres
+```
+
+You need one connection string for your development database and one for your test database. You can read more info about it [here](https://redwoodjs.com/docs/testing#the-test-database).
+
+Edit the `scripts/seed.ts` file and uncomment the contents of the array that contain the "fake" users. We will also use the `createMany` method for inserting records in the data base so we can skip the duplicates [more info](<https://www.prisma.io/docs/reference/api-reference/prisma-client-reference#createmany>). It should look like this:
+
+```ts
+    ...
```
[the remark this answers is not in the document]
Comment (jtoar) [reply docs/docs/how-to/test-in-github-actions.md:117]: @esteban-url nah this should be ok; see this comment here https://github.com/redwoodjs/redwood/pull/6921#issuecomment-1346800965

**Comment by esteban-url** [comment]
Thanks! That makes a lot of sense. I'll keep it in mind for my next docs PR!

**Comment by jtoar** [comment]
@esteban-url reading through now and making some edits I'll push up as I go—great stuff! I'll focus on getting this merged today.

**Comment by esteban-url** [comment]
Thanks! yeah, feel free to edit anything.

**Comment by jtoar** [comment]
Again, great stuff @esteban-url! And thanks for letting me have at it a bit. The diff of edits may look large, but it was just polishing, made easier by the great structure that was already there. A few tips for the future:

- use relative links for other redwood docs. it lets docusaurus do it's thing 
- for code blocks, you can use a title "attribute" as the de facto place for the the file's title

**Review by jtoar — APPROVED** [review]

**Pull request merged**